tests/streamlit_ui_tests/graph.spec.ts

```diff
@@ -44,13 +44,41 @@ const testDeleteTitleThroughContextMenu = async (page, selector) => {
     await expect(mito.locator(selector)).not.toBeVisible();
 };
 
+const addColumnToAxis = async (mito: FrameLocator, page: Page, axis: 'X' | 'Y', columnName: string) => {
+  await mito.locator('.spacing-row', { hasText: `${axis} axis` }).locator('.mito-dropdown-button').click();
+  await mito.locator('.mito-dropdown-item', { hasText: columnName }).click();
+  await awaitResponse(page);
+}
+
+const openGraphEditor = async (mito: FrameLocator, page: Page) => {
+  await clickButtonAndAwaitResponse(page, mito, { name: 'Graph' })
+  await expect(mito.locator('.g-gtitle')).toBeVisible();
```

Comment:
This would only pass if the before calling `openGraphEditor` you have an entire column selected, right? Because otherwise we won't generate a graph and the g-gittle won't be visible. 

Maybe we should just remove the check and add an `await awaitResponse(page);`

Reply:
I tried that but the way that we open the graph displays stuff and doesn't include a visual loading indicator, so it registers as finished before it actually loads the graph. This causes inconsistent behavior in the tests. You're right that a column needs to be selected but I think given that the tests are generally doing that it's ok. I could put it in a wrapper with opening a specific csv so that it's guaranteed that a column is selected?

Reply:
Okay makes sense. Want to just add this as a comment in the utility so if we ever run into an issue with it we have this context to help us debug?